public/js/searchbar.js

```diff
@@ -0,0 +1,191 @@
+"use strict";
+const sqlAuthentication = { // sql connection settings
+  host: "127.0.0.1", // for Mac os, type 127.0.0.1
+  user: "root",
+  password: "",
+  multipleStatements: true,
+  database: "COMP2800" // Database name
+}
+
+const userTable = "BBY_19_user"; // Table name
+
+ready(async function () {
+  async function postData(url, data) {
+    const response = await fetch(url, {
+      method: 'POST',
+      mode: 'same-origin',
+      cache: 'default',
+      credentials: 'same-origin',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Requested-With': 'XMLHttpRequest'
+      },
+      redirect: 'follow',
+      referrerPolicy: 'no-referrer',
+      body: JSON.stringify(data)
+    });
+    return response.json();
+  }
+
+  document.getElementById("searchButton").addEventListener("click", async function (e) {
+    var keyword = document.getElementById("searchKeyword").value;
+    let response = await postData("/getUsersKeyword", {
+      keyword: keyword
+    });
+
+    if (response.users.length == 0) {
+      alert("No search results found.");
```

Comment:
Aaron says we shouldn't use alerts. "Do not use the alert in JS ... for anything. Ever. Period."

Reply:
Okay, I'll delete it.

Reply:
Replaced the alert to message warning below the searchbar.